megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -669,180 +715,197 @@ private void showRoundReport() {
-            ignoreHotKeys = true;
-            boardSaveAsImage(true);
-            ignoreHotKeys = false;
-        } else if ("fileBoardSaveAsImageUnits".equalsIgnoreCase(event.getActionCommand())) { //$NON-NLS-1$
-            ignoreHotKeys = true;
-            boardSaveAsImage(false);
-            ignoreHotKeys = false;
-        } else if ("replacePlayer".equalsIgnoreCase(event.getActionCommand())) { //$NON-NLS-1$
-            replacePlayer();
-        } else if (event.getActionCommand().equals(VIEW_MEK_DISPLAY)) {
-            toggleDisplay();
-        } else if (event.getActionCommand().equals(VIEW_ACCESSIBILITY_WINDOW)) {
-            toggleAccessibilityWindow();
-        } else if (event.getActionCommand().equals(VIEW_MINI_MAP)) {
-            toggleMap();
-        } else if (event.getActionCommand().equals(VIEW_UNIT_OVERVIEW)) {
-            toggleUnitOverview();
-        } else if (event.getActionCommand().equals(VIEW_LOS_SETTING)) {
-            showLOSSettingDialog();
-        } else if (event.getActionCommand().equals(VIEW_ZOOM_IN)) {
-            bv.zoomIn();
-        } else if (event.getActionCommand().equals(VIEW_ZOOM_OUT)) {
-            bv.zoomOut();
-        } else if (event.getActionCommand().equals(VIEW_TOGGLE_ISOMETRIC)) {
-            GUIPreferences.getInstance().setIsometricEnabled(
-                    bv.toggleIsometric());
-        } else if (event.getActionCommand().equals(VIEW_TOGGLE_FOV_HIGHLIGHT)) {
-            GUIPreferences.getInstance().setFovHighlight(
-                    !GUIPreferences.getInstance().getFovHighlight());
-            bv.refreshDisplayables();
-            if (client.getGame().getPhase() == Phase.PHASE_MOVEMENT) {
-                bv.clearHexImageCache();
-            }
-        } else if (event.getActionCommand().equals(VIEW_TOGGLE_FIELD_OF_FIRE)) {
-            GUIPreferences.getInstance().setShowFieldOfFire(
-                    !GUIPreferences.getInstance().getShowFieldOfFire());
-            bv.repaint();
-        } else if (event.getActionCommand().equals(VIEW_TOGGLE_FOV_DARKEN)) {
-            GUIPreferences.getInstance().setFovDarken(
-                    !GUIPreferences.getInstance().getFovDarken());
-            bv.refreshDisplayables();
-            if (client.getGame().getPhase() == Phase.PHASE_MOVEMENT) {
-                bv.clearHexImageCache();
-            }
-        } else if (event.getActionCommand()
-                .equals(VIEW_TOGGLE_FIRING_SOLUTIONS)) {
-            GUIPreferences.getInstance().setFiringSolutions(
-                    !GUIPreferences.getInstance().getFiringSolutions());
-            if (!GUIPreferences.getInstance().getFiringSolutions()) {
-                bv.clearFiringSolutionData();
-            } else {
-                if (curPanel instanceof FiringDisplay) {
-                    ((FiringDisplay) curPanel).setFiringSolutions();
+        switch (event.getActionCommand()) {
+            case FILE_GAME_SAVE:
```

Comment:
I'll assume you kept the internal logic the same and just changed the if/else block to a switch statement.

Reply:
Exactly